lib/make_channel_for_archive.rb

```diff
@@ -0,0 +1,81 @@
+# frozen_string_literal: true
```

Comment:
Ignore this whole file for review, but if you want to test this archiving functionality you can use this to make topics with a channel and 1000 messages:

`DiscourseChat::MakeChannelForArchive.run("martin")`

Reply:
I am not really sure where to put this yet, I just committed it so I could move between my PC and laptop.